File: HomeStorage.Logic/Models/ProductModels/ProductModel.cs
namespace HomeStorage.Logic.Models.ProductModels
{
    public class ProductModel
    {
        public required Guid ProductId { get; set; }
        public required Guid LocationId { get; set; }
        public required string Name { get; set; }
        public string? Description { get; set; }
        public string? ImageUrl { get; set; }
        public List<LookupModel<Guid>> Categories { get; set; } = [];
        public DateTime? ExpirationDate { get; set; }
        public double? Amount { get; set; }
    }
}

File: frontend/src/models/product/productModel.ts
import { Lookup } from "@/models/lookup";

export interface ProductModel {
    productId: string;
    locationId: string;
    name: string;
    description?: string;
    imageUrl?: string;
    categories: Lookup<string>[];
    expirationDate?: string;
    amount?: number;
}
